recipe/meta.yaml

```diff
@@ -5,7 +5,7 @@ package:
   version: {{ version }}
 
 build:
-  number: 4
+  number: 5
```

Comment:
Bumping the build number will cause it to upload new (though identical) binaries to anaconda.org for all the other platforms. Not the end of the world, but it will take up storage space

Reply:
@jdblischak if I keep the build number the same your saying it doesn't upload already existing packages? Didn't realize that, awesome! I'll change it back.